main.py

```diff
@@ -1,10 +1,47 @@
 import init_django_orm  # noqa: F401
 
+import json
+
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    Race.objects.all().delete()
+    Skill.objects.all().delete()
+    Guild.objects.all().delete()
+    Player.objects.all().delete()
+
+    with open("players.json", "r") as f:
+        data = json.load(f)
+
+    players = data
+    for player in players:
```

Comment:
Here you can immediately iterate over both the key and the value, which will simplify your code a bit.